esmvalcore/preprocessor/_time.py

```diff
@@ -460,7 +460,9 @@ def anomalies(cube, period, reference=None, standardize=False):
         reference_cube = extract_time(cube, **reference)
     reference = climate_statistics(reference_cube, period=period)
     if period in ['full']:
-        cube = cube - reference
+        orig_cube = cube.copy()
+        cube = orig_cube - reference
+        cube.metadata = orig_cube.metadata
```

Comment:
```suggestion
        metadata = copy.deepcopy(cube.metadata)
        cube = cube - reference
        cube.metadata = metadata
```
would probably be a bit more efficient because it only copies the required metadata instead of the entire cube

Reply:
yer, I did think about that too but the cube is lazy anyway so it's not too heavy on the memory; this way we are totally destroying the original cube but it looks like it's not needed anymore anyway